push-api/JSONParser/config.yml

```diff
@@ -1,9 +1,11 @@
 headers:
-  apiKey: "Bearer <access token>"
-  organisation: "<organisation name>"
-  sourceName: "<source name>"
-  url: "<node containing the url>"
+  apiKey: "Bearer <Replace with Api key>"
+  organisation: "<Replace with org name>"
+  sourceName: "<Replace with org source>"
+  url: "<replace with>" #The node name of a single document URL inside the json document array (default : "url")
 
 basicInformation:
-  array: "<array containing the documents>"
-  filePath: "<file path>"
\ No newline at end of file
+  array: "<replace with node name>" #the main node of the json file (default : "documents")
```

Comment:
Some spaces before semi-colons here. Remove them all :) 
& JSON has to be upper case.